teku/src/main/java/tech/pegasys/teku/cli/options/DepositOptions.java

```diff
@@ -21,7 +21,8 @@
   @Option(
       names = {"--eth1-deposit-contract-address"},
       paramLabel = "<ADDRESS>",
-      description = "Contract address for the deposit contract. Required if eth1-endpoint is set.",
+      description =
+          "Contract address for the deposit contract. Generally defaults if network has been specified.",
```

Comment:
hmm, not sure "Generally" is a particularly good term here. Maybe:
```suggestion
          "Contract address for the deposit contract. Only required when creating a custom network.",
```